documentation/src/docs/pravega-concepts.md

```diff
@@ -362,3 +360,13 @@ Tier 2 Storage provides a highly-scalable, high-throughput cost-effective
 storage. We expect this Tier 2 to be typically deployed on spinning disks. Pravega
 asynchronously migrates Events from Tier 1 to Tier 2 to reflect the different
 access patterns to Stream data. Tier 2 Storage is based on an HDFS model. 
+
+### Retention Policy
```

Comment:
Maybe the title could be: `Stream Retention Policies`.

Reply:
Accepted suggestion